### PR TITLE
Adding argo-server, argo-workflow-controller and argo-executor to test-tools

### PR DESCRIPTION
**Pull request by rajdas98**
**What this PR does / why we need it**:
Adding argo-server, argo-workflow-controller and argo-executor to test-tools

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: custom/argo-workflow-executor/buildscripts/push [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -e
+
+IMAGEID=$( docker images -q litmuschaos/argoexec )
+
+if [ ! -z "${DNAME}" ] && [ ! -z "${DPASS}" ];
+then
+  docker login -u "${DNAME}" -p "${DPASS}";
+  #Push to docker hub repository with latest tag
```
Comment (uditgaurav) [inline custom/argo-workflow-executor/buildscripts/push:9]: do we need this comment in all files? we are not pushing with latest tag.
OR should we push with the latest tag?

**Comment by uditgaurav** [comment]
should we also add for mongo?

File: custom/argo-workflow-executor/buildscripts/push (hunk 1 above)
Comment (rajdas98) [reply custom/argo-workflow-executor/buildscripts/push:9]: It's already there in buildscript/push

**Comment by rajdas98** [comment]
> should we also add for mongo?

Done

**Review by uditgaurav — APPROVED** [review]

**Review by oumkale — APPROVED** [review]

**Pull request merged**